packages/auth/src/mfa/assertions/totp.test.ts

```diff
@@ -208,6 +213,78 @@ describe('core/mfa/totp/assertions/TotpMultiFactorAssertionImpl', () => {
   });
 });
 
+describe('Testing signIn Flow', () => {
+  let auth: TestAuth;
+  let assertion: MultiFactorAssertionImpl;
+  let totpSignInResponse: StartTotpMfaSignInResponse;
+  let session: MultiFactorSessionImpl;
+  beforeEach(async () => {
+    mockFetch.setUp();
+    auth = await testAuth();
+    session = MultiFactorSessionImpl._fromMfaPendingCredential(
+      'mfa-pending-credential'
+    );
+  });
+  afterEach(mockFetch.tearDown);
+
+  it('should finalize mfa signin for totp', async () => {
+    totpSignInResponse = { 'verificationCode': '123456' } as any;
+    assertion = TotpMultiFactorGenerator.assertionForSignIn(
+      'enrollment-id',
+      '123456'
+    ) as any;
+
+    const mock = mockEndpoint(
+      Endpoint.FINALIZE_MFA_SIGN_IN,
+      totpSignInResponse
```

Comment:
Finalize needs to return id and refresh tokens - https://cloud.google.com/identity-platform/docs/reference/rest/v2/accounts.mfaSignIn/finalize, https://github.com/firebase/firebase-js-sdk/blob/04dcdbb0f5b19e51082a846463d84aadb77e98d4/packages/auth/src/api/authentication/mfa.test.ts#L115

 Let's use that same format in the mocks.